src/Base/BaseAttributes.php

```
@@ -206,6 +207,48 @@ public static function render(array $attributes): string
+     * (`prefixkey`).
+     *
+     * @return array Associative array of expanded attribute key-value pairs.
+     *
+     * @phpstan-param mixed[] $values
+     * @phpstan-return array<string, string>
+     */
+    private static function expandPrefixedAttributes(
+        array $values,
+        string $prefix,
+        bool $encode,
+        bool $dashedPrefix = true,
```

PHPMD (2.15.0)

[error] The method expandPrefixedAttributes has a boolean flag argument $dashedPrefix, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)